src/adapter/http/tests/harness/server_side_harness.rs

```diff
@@ -44,7 +44,7 @@ pub(crate) trait ServerSideHarness {
         self.dataset_url_with_scheme(dataset_alias, "odf+http")
     }
 
-    fn system_time_source(&self) -> &SystemTimeSourceStub;
+    fn system_time_source(&self) -> &FakeSystemTimeSource;
```

Comment:
I was talking about keeping `SystemTimeSourceStub` for simple cases we had before. It would have only:
  - set logic
  - unset logic
  - default constructor with current time
  
While the advance fake version would be another implementation struct that we will only use in flow service tests for now. And that should not contain free default time, always starting from fixed test time and advancing via `sleep` method only.

Reply:
Yes, thanks for the comment, I misunderstood last time -- done for now